actions/handlers/twap.ts

```diff
@@ -0,0 +1,14 @@
+import { TWAP_ADDRESS } from "@cowprotocol/cow-sdk";
+
+import { SmartOrderValidator, ValidationResult } from "../model";
+
+export const handlerAddress = TWAP_ADDRESS;
+
+export const validateOrder: SmartOrderValidator = async (_params) => {
+  // TODO: Implement validation logic, ideally delegate to the SDK
```

Comment:
This PR don't implement the specic logic, this could come in future PRs. 

As hinted here, most of the logic would ideally come from the SDK. This validator would just transform it in the result that this function expects